web/hosts.go

```diff
@@ -15,6 +15,28 @@ import (
 	"github.com/trento-project/trento/internal/sapsystem"
 )
 
+type HealthContainer struct {
+	Passing  int
+	Warning  int
+	Critical int
+	Layout   string
+}
+
+func NewHealthContainer(hostList hosts.HostList) *HealthContainer {
+	h := &HealthContainer{}
```

Comment:
Maybe we could move the `HealthContainer` to a new file and rename this to `NewHostsHealthContainer`  or use an `[]interface` with `.Health()` as constructor parameter so that we can use the same logic in other pages as well.

Reply:
Ok, if this helps for reusability I will move it to a new file and apply these changes

Reply:
@fabriziosestito Changes done in: https://github.com/trento-project/trento/pull/139/commits/f5612982eb924c4c2efacaa903579291a527f7aa

Let's go with this easy approach by now.

If you agree, we can merge afterwards